webapp/components/more_channels.jsx

```diff
@@ -33,6 +36,8 @@ export default class MoreChannels extends React.Component {
         this.handleNewChannel = this.handleNewChannel.bind(this);
         this.createChannelRow = this.createChannelRow.bind(this);
 
+        this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate.bind(this);
```

Comment:
Technically this one isn't pure because it depends on render height. However it is already broken so I think it is fine.